.github/workflows/ci.yml

```diff
@@ -58,7 +57,7 @@ jobs:
         working-directory: ./server
 
       - name: 💻 Install and build frontend
-        run: docker-compose run --rm yarn install && docker-compose run --rm yarn build-local
+        run: docker-compose run --rm yarn install && yarn build-local
```

Comment:
Wollen wir dann yarn nicht überall ersetzen?

Die Zeiten das Yarn wirklich besser was als NPM sind auch vorbei.

Reply:
Hatte ich, aber in der Client CI bei den Tests gab es einen Zugriffsrechte Bug, den ich nicht gefixt bekommen habe. Dann hab ich wieder yarn genommen.